### DISCUSSION TITLE
Post panel

### DISCUSSION THREAD

**Discussion by wildserver** [razonyang/hugo-theme-bootstrap#986] [General]
Hello. While adapting it to suit myself and my language, I noticed one inconvenience - the lack of multilingual support for the “Post Panel” for some elements.

I had to create my custom files in `/myblog/layouts/partials/post/panel`

```markdown
comments.html
copyright.html
sidebar-toggler.html
toc.html
translations.html
```

Agree, it would be more convenient if you correct the files `/hugo-theme-bootstrap/layouts/partials/post/panel` in the theme source by changing to `title="{{ i18n "text" }}` in the above files, like this:

`comments.html`

```html
...
<a class="action action-post-comments" href="#post-comments" role="button" aria-label="Comments" title="{{ i18n "comments" }}>
...
```

`copyright.html`

```html
...
<a class="action action-copyright" href="#post-copyright" role="button" aria-label="Copyright" title="{{ i18n "copyright" }}">
...
```

`sidebar-toggler.html`

```html
...
<a id="sidebarToggler" class="action action-sidebar-toggler d-none d-xxl-block" role="button" title="{{ i18n "sidebar_toggler" }}">
...
```

`toc.html`

```html
...
<a class="action action-toc d-none d-xxl-block" href="#postTOC" role="button" title="{{ i18n "table_of_contents" }}">
...
<a class="action action-toc d-block d-xxl-none" href="#post-toc-container" role="button" title="{{ i18n "table_of_contents" }}">
...
```

`translations.html`

```html
...
title="{{ i18n "translations" }}">
...
```

And add the following lines to `themes/hugo-theme-bootstrap/i18n/en.toml`:

```markdown
[sidebar_toggler]
other = "Sidebar toggler"

[translations]
other = "Translations"
```

Then there will be no difficulties with translation into other languages.

**Comment by razonyang** [comment]
Will fix later, PR are also welcome.

**Comment by razonyang** [comment]
Closed by #987
